openseespy-pip/openseespy/postprocessing/Get_Rendering.py

```diff
@@ -585,29 +561,17 @@ def nodecoordsEigen(nodetag):
 				ipltf._plotCubeVol(iNode_final, jNode_final, kNode_final, lNode_final, iiNode_final, jjNode_final, kkNode_final, llNode_final, 
 								ax, show_element_tags, eleTag, "solid", fillSurface='yes')
 								
-		nodeMins = np.array([min(DeflectedNodeCoordArray[:,0]),min(DeflectedNodeCoordArray[:,1]),min(DeflectedNodeCoordArray[:,2])])
-		nodeMaxs = np.array([max(DeflectedNodeCoordArray[:,0]),max(DeflectedNodeCoordArray[:,1]),max(DeflectedNodeCoordArray[:,2])])
-				
-		xViewCenter = (nodeMins[0]+nodeMaxs[0])/2
-		yViewCenter = (nodeMins[1]+nodeMaxs[1])/2
-		zViewCenter = (nodeMins[2]+nodeMaxs[2])/2
-		
-		view_range = max(max(DeflectedNodeCoordArray[:,0])-min(DeflectedNodeCoordArray[:,0]), 
-							max(DeflectedNodeCoordArray[:,1])-min(DeflectedNodeCoordArray[:,1]), 
-							max(DeflectedNodeCoordArray[:,2])-min(DeflectedNodeCoordArray[:,2]))
-
-		ax.set_xlim(xViewCenter-(view_range/2), xViewCenter+(view_range/2))
-		ax.set_ylim(yViewCenter-(view_range/2), yViewCenter+(view_range/2))
-		ax.set_zlim(zViewCenter-(view_range/2), zViewCenter+(view_range/2))
 		ax.text2D(0.10, 0.95, "Mode "+str(modeNumber), transform=ax.transAxes)
 		ax.text2D(0.10, 0.90, "T = "+str("%.3f" % Tn)+" s", transform=ax.transAxes)
+
 				
+	ipltf._setStandardViewport(fig, ax, DeflectedNodeCoordArray, len(nodecoords(nodetags[0])))
 	plt.axis('on')
 	plt.show()
-	return fig
+	return fig, ax
 	
 
-def plot_deformedshape(Model="none", LoadCase="none", tstep = -1, scale = 200, overlap='no'):
+def plot_deformedshape(Model="none", LoadCase="none", tstep = -1, scale = 1, overlap='no'):
```

Comment:
I will make this change for now.

I would argue it depends on the system being analyzed, so it might be better to allow the user to control the scale for their displacement. Doing otherwise might lead to confusion.

Reply:
I think we should have some default value here. If the users feel that the default scale factor is not suitable to their use, they can change it manually.